apollo/index.js

```diff
@@ -78,7 +77,4 @@ const server = new ApolloServer({
   playground: true,
 });
 
-const app = express();
-server.applyMiddleware({ app, path: '*' });
-
-module.exports = app;
+module.exports = apolloServer.createHandler({ path: '*' });
```

Comment:
I tried to run this locally but `apolloServer` is not defined. Did you mean this?
```suggestion
module.exports = ApolloServer.createHandler({ path: '*' });
```

Reply:
oh ya 🤭was working too quickly. changing that now

Reply:
I tried to deploy this and it doesn't work with the path set either, so the following would be the method to use: 
```suggestion
module.exports = server.createHandler();
```